src/data-types/strings.js

```diff
@@ -6,16 +6,16 @@ const secondName = 'Smith'
 // TODO: Update the code using Javascript string operations and the variables above so that the tests pass.
 
 // Set this variable to firstName and secondName concatenated
-const fullName = null
+const fullName = 'Jane Smith'
 
 // Set this variable to the 10th character of the alphabet variable
-const tenthCharacterOfAlphabet = null
+const tenthCharacterOfAlphabet = alphabet[9]
```

Comment:
This is great. This is the same method we were looking for the above comments. I.E getting the voucher code/city names